sdks/python/apache_beam/ml/inference/pytorch_inference_test.py

```diff
@@ -38,6 +38,8 @@
   from apache_beam.ml.inference.api import PredictionResult
   from apache_beam.ml.inference.base import RunInference
   from apache_beam.ml.inference.pytorch_inference import PytorchModelHandler
```

Comment:
For the tests, I changed them to use PytorchModelHandlerTensor, and PytorchModelHandlerKeyedTensor. Should the tests default to use `PytorchModelHandler` since PytorchModelHandlerTensor, and PytorchModelHandlerKeyedTensor are experimental? How should we handle this

Reply:
Let's drop `PytorchModelHandler` and exercise the type-specific ones.